layouts/shortcodes/observability_pipelines/processors/quota.md

```diff
@@ -5,8 +5,12 @@ As an example, you can configure this processor to drop new logs or trigger an a
 To set up the quota processor:
 1. Enter a name for the quota processor. The pipeline uses the name to identify the quota across multiple Remote Configuration deployments of the Worker.
 1. Define a **filter query**. Only logs that match the specified [filter query](#filter-query-syntax) are counted towards the daily limit.
+    - Logs that match the quota filter and are within the daily quota, are counted towards the daily limit and sent to the next step in the pipeline.
```

Comment:
Thoughts on this?
All logs that match the quota filter are counted towards the daily limit and only those within the daily quota limit are sent to the next step in the pipeline.

Reply:
Actually is line 7 and 8 overlapping?

Reply:
DM'd. All good here.